scrapy/pipelines/media.py

```diff
@@ -126,37 +123,6 @@ def _process_request(self, request, info, item):
         )
         return dfd.addBoth(lambda _: wad)  # it must return wad at last
 
-    def _make_compatible(self):
-        """Make overridable methods of MediaPipeline and subclasses backwards compatible"""
-        methods = [
-            "file_path",
-            "thumb_path",
-            "media_to_download",
-            "media_downloaded",
-            "file_downloaded",
-            "image_downloaded",
-            "get_images",
-        ]
-
-        for method_name in methods:
-            method = getattr(self, method_name, None)
-            if callable(method):
-                setattr(self, method_name, self._compatible(method))
-
-    def _compatible(self, func):
```

Comment:
Removed as well because is not called anymore after  `_make_compatible` removal.

Reply:
Yeah, this was the intention.